services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/controller/UserGroupController.java

```diff
@@ -105,6 +132,11 @@ public ResponseEntity<List<UserGroup>> getAllUserGroupByGroupId(
   }
 
   @GetMapping("{id}/user-groups/count")
+  @PreAuthorize(
+    "isAuthenticated() && hasAnyAuthority('" +
```

Comment:
```suggestion
    "isAuthenticated() && hasAuthority('" +
```

Reply:
Has any authority for multiple permission case, where the api can run through either of the permissions provided